src/components/SearchBar/index.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+import { teamDetails } from "../../assests/teamDetails.js";
+import "./style.css";
+
+const onSearchTeamDetails = (searchText, teamDetails) => {
+  return teamDetails.filter(
+    (member) =>
+      member.name.toLowerCase().includes(searchText) ||
```

Comment:
There is a bug here. You have to make your search text also lowercase before comparing. What if someone searches for "Upaharika"? and "upaharika" doesn't match?